### PR TITLE
Drop jQuery, fix a number of XSS

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### CODE REVIEW

**Comment by dgrammatiko** [comment]
ping @roland-d

**Comment by roland-d** [comment]
@dgrammatiko Awesome, it is on my todo list but no time yet. Will review later.

**Comment by dgrammatiko** [comment]
fwiw I just converted the code, haven't tested it 😅

**Comment by roland-d** [comment]
@dgrammatiko That is fine, that is why I did not merge it for this release :P Will review and test.

File: media/com_patchtester/js/fetcher.js [hunk 1]
```diff
@@ -1,134 +1,112 @@
...
+
+          if (response.data.header) {
+            document.getElementById('patchtester-progress-header').innerHTML = Joomla.sanitizeHtml(response.data.header);
+          }
+
+          if (!response.data.complete) {
+            // Send another request
+            this.request('fetch');
+          } else {
+            document.getElementById('rogress').remove();
```
Comment (heelc29) [inline media/com_patchtester/js/fetcher.js:81]:
```suggestion
            document.getElementById('progress').remove();
```

**Comment by roland-d** [comment]
@heelc29 I am currently reviewing/updating this PR.

**Comment by roland-d** [comment]
@dgrammatiko I have tested this and there were quite a few things not working but all should be fixed now. Can you give it a check to see if there is nothing obvious wrong. Thanks.

Indexing the pull requests works now :)

**Comment by dgrammatiko** [comment]
LGTM, sorry for the silly mistakes 😀

**Pull request merged**